Makefile

```diff
@@ -1,22 +1,23 @@
 PREFIX ?= /usr
+DESTDIR ?=
 BINDIR ?= $(PREFIX)/bin
 LIBDIR ?= $(PREFIX)/share/hilbish
 
 build:
 	@go build
 
-install:
-	@install -v -d "$(BINDIR)/" && install -m 0755 -v hilbish "$(BINDIR)/hilbish"
-	@mkdir -p "$(LIBDIR)"
-	@cp libs preload.lua .hilbishrc.lua "$(LIBDIR)" -r
-	@echo /usr/bin/hilbish >> /etc/shells
+install: build
+	@install -v -d "$(DESTDIR)$(BINDIR)/" && install -m 0755 -v hilbish "$(DESTDIR)$(BINDIR)/hilbish"
+	@mkdir -p "$(DESTDIR)$(LIBDIR)"
+	@cp libs preload.lua .hilbishrc.lua "$(DESTDIR)$(LIBDIR)" -r
+	@echo "$(DESTDIR)$(BINDIR)/hilbish" >> /etc/shells
 	@echo "Hilbish Installed"
 
 uninstall:
 	@rm -vrf \
-			"$(BINDIR)/hilbish" \
-			"$(LIBDIR)"
-	@sed '/\/usr\/bin\/hilbish/d' /etc/shells
+			"$(DESTDIR)$(BINDIR)/hilbish" \
+			"$(DESTDIR)$(LIBDIR)"
+	@sed -i '/$(DESTDIR)\/usr\/bin\/hilbish/d' /etc/shells
```

Comment:
We can probably simplify this to `@sed -i '/$(DESTDIR)$(BINDIR)\/hilbish/d' /etc/shells`

Reply:
True

Reply:
:(

Reply:
Actually `@sed -i '/hilbish/d' /etc/shells` will work fine